pass_import.py

```diff
@@ -525,6 +526,54 @@ class Dashlane(PasswordManagerCSV):
             'url': 'url', 'comments': 'comments'}
 
 
+class Encryptr(PasswordManagerCSV):
+    # Looking at how Encryptr processes exports here (method getCsvFields):
+    # https://github.com/SpiderOak/Encryptr/blob/master/src/views/MainView.js#L163
+    # and entry types contained here:
+    # https://github.com/SpiderOak/Encryptr/tree/master/src/models/types
+    # we conclude:
+    # Every record has metadata: "Entry Type","Label"
+    # Every record happens to have, independently: "Notes"
+    # Password type has: "Username","Password", "Site URL"
+    # General type has: "Text"
+    # Credit card type has: "Type", "Name on card", "Card Number", "CVV", "Expiry"
+    # encryptr dynamically generates CSVs based on available entry types,
+    # and supports credit cards.
+    # all told, we have, in no particular order as it's dynamic:
+    # "Entry Type","Label", "Notes", "Username", "Password", "Site URL",
+    # "Text", "Type", "Name on card", "Card Number", "CVV", "Expiry"
+
+    # and now let's map it all. Ignoring Entry Type.
+    keyslist = [
```

Comment:
You don't need to overwrite the keylist here. This list simply provide a list of classic entry that will be imported without the -e option. It usually does not change through the importer. The dict key is enough to create the key mapping.